internal/gossip/gossip.go

```diff
@@ -43,6 +43,7 @@ func (g *InMemoryGossip) Publish(channel string, value []byte) error {
 		return errors.New("gossip has been stopped")
 	case g.channel <- msg.ToBytes():
 	default:
+		// drop the message if the channel is full
```

Comment:
Logging when we do this would be helpful

Reply:
I will, but prefer to wait until #1159 is merged before I touch gossip code. This was me just wanting to note that this could drop data.